Firmware/LightpackUSB.c

```diff
@@ -211,6 +212,19 @@ void CALLBACK_HID_Device_ProcessHIDReport(USB_ClassInfo_HID_Device_t* const HIDI
     case CMD_NOP:
         break;
 
+
+    // Unofficial commands
+    case CMD_UNOFFICIAL_SET_USBLED:
+
+        g_Settings.isUsbLedEnabled = ReportData_u8[1];
+        if (g_Settings.isUsbLedEnabled) {
+            SET(USBLED);
+        } else {
+            CLR(USBLED);
+        }
+
+        break;
+
```

Comment:
No, they are not persistent.
That may not really be a problem, because whenever Prismatik is started it writes all settings to the device.
When the device is disconnected Prismatik will also write the settings when the device is reconnected again.